packages/paste-core/components/help-text/__tests__/helptext.test.tsx

```diff
@@ -21,3 +26,83 @@ describe('HelpText marginTop prop', () => {
     expect(container.find('Flex').prop('marginTop')).toEqual('space0');
   });
 });
+
+describe('HelpText HTML attributes', () => {
+  it('should set element data attribute for flex wrapper and help text', (): void => {
+    const {container} = render(<HelpText data-testid="help_text">This is help text.</HelpText>);
+    expect(container.querySelector('[data-paste-element="HELP_TEXT_WRAPPER"]')).toBeInTheDocument();
```

Comment:
This might be a nit pick, but I wonder if it is more useful to check not only for the existence for the correct `paste-element` data attributes, but also that they are on the correct elements. That would mean that we should find the target nodes in the rendered tree, and confirm that the attribute is on the correct node.
The extra constraint for the test potentially gives us more security against future regressions.

Reply:
@andipants12 would the test on line 35 (`expect(screen.getByTestId('help_text').getAttribute('data-paste-element')).toEqual('HELP_TEXT');` be more secure in the way you're describing? I've also refactored this component so that it only needs one `element` attribute, so this Wrapper-specific one will be soon deleted.